app/assets/javascripts/views/travel-advice.js

```diff
@@ -147,14 +148,11 @@
     }
   }
 
-  CountryFilter.prototype.pagePath = function () {
-    window.location.pathname.split('/').pop()
```

Comment:
This function is not used anywhere and it cannot work, because it doesn't do anything (the "return" is missing). It also doesn't have any tests.

Reply:
Removal of this function has been extracted to a separate PR.